news/2021-03-02-mappers-report-february.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "The Mappers' Report: February"
+date: 2021-03-02 08:00:00 +0000 
+---
+
+News news and even more news! The February issue of the Mappers' Report is here to inform you about everything mapping-related that happened last month!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains the following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers' Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+More ranking criteria changes! And we're here to show you them all! Actually, there was just one this month. However, we have new open proposals which need more input from the community, so head right over to the ranking criteria forum and drop your opinions!
```

Comment:
```suggestion
There's been more ranking criteria changes, and we're here to show you them all! Actually, there was just one this month. 

However, we have new open proposals which need more input from the community, so head right over to the posts listed below and make your opinions heard.
```

also fwiw, be careful not to overuse exclamations - they tend to be best for end of sentence emphasis or tone and rarely used elsewhere